c/validation/adbc_validation_util.h

```diff
@@ -31,6 +31,7 @@
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
 #include <nanoarrow/nanoarrow.h>
+#include <utils.h>
```

Comment:
Best to use "" includes for our headers

Hmm, we may also want to switch to `common/utils.h` or something eventually, a bare utils.h is liable to get mixed up